packages/wrangler/src/dev/miniflare.ts

```diff
@@ -360,7 +360,13 @@ export function buildMiniflareBindingOptions(config: MiniflareBindingsConfig): {
 	}
 
 	const bindingOptions = {
-		bindings: bindings.vars,
+		bindings: {
+			...bindings.vars,
+			// emulate version_metadata binding via a JSON var
+			...(bindings.version_metadata
+				? { [bindings.version_metadata.binding]: { id: randomUUID(), tag: "" } }
```

Comment:
This feels like it should be in miniflare proper—why did you choose to emulate it with a JSON binding?